src/app.js

```diff
@@ -0,0 +1,260 @@
+function updateIcon(iconElement, icon, description) {
+  const icons = {
+    "01d": "clear-sky-day.svg",
+    "01n": "clear-sky-night.svg",
+    "02d": "few-clouds-day.svg",
+    "02n": "few-clouds-night.svg",
+    "03d": "scattered-clouds.svg",
+    "03n": "scattered-clouds.svg",
+    "04d": "broken-clouds.svg",
+    "04n": "broken-clouds.svg",
+    "09d": "shower-rain.svg",
+    "09n": "shower-rain.svg",
+    "10d": "rain-day.svg",
+    "10n": "rain-night.svg",
+    "11d": "thunderstorm.svg",
+    "11n": "thunderstorm.svg",
+    "13d": "snow.svg",
+    "13n": "snow.svg",
+    "50d": "mist-day.svg",
+    "50n": "mist-night.svg",
+  };
+
+  iconElement.setAttribute("alt", description);
+
+  iconElement.setAttribute("src", `images/icons/${icons[icon]}`);
+}
+
+function showCurrentDate(element) {
+  let currentDate = new Date();
+
+  let weekDays = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+  ];
+
+  let day = weekDays[currentDate.getDay()];
+  let hours = currentDate.getHours();
+  let minutes = currentDate.getMinutes();
+
+  if (hours < 10) {
+    hours = `0${hours}`;
+  }
+
+  if (minutes < 10) {
+    minutes = `0${minutes}`;
+  }
```

Comment:
A simpler way to get the 2 digit format of the hour/minute. 💅 

```suggestion
  let hours = ("0" + currentDate.getHours()).slice(-2);
  let minutes = ("0" + currentDate.getMinutes()).slice(-2);
```